src/digest/digest.rs

```diff
@@ -27,7 +27,7 @@
 #![allow(unsafe_code)]
 
 use {c, init, polyfill};
-use core;
+use core::{self, fmt};
```

Comment:
Please use just `use core;` and qualify `core::fmt`. See https://github.com/briansmith/ring/blob/master/STYLE.md#imports-use-and-qualification.